src/hooks/useOptions.ts

```diff
@@ -8,10 +8,11 @@ export enum UnityPreviewMode {
   MARKETPLACE = 'marketplace',
   AUTHENTICATION = 'authentication',
   BUILDER = 'builder',
+  CONFIGURATOR = 'configurator',
 }
 
 export interface OptionsWithSource {
-  options: PreviewOptions & { mode: UnityPreviewMode | null; disableLoader: boolean }
+  options: PreviewOptions & { mode: UnityPreviewMode | null; disableLoader: boolean; username: string | null }
```

Comment:
wdyt about creating a new type like `PreviewOptions` with these props?

```
import { PreviewOptions as BasePreviewOptions } from '@dcl/schemas'

export type PreviewOptions = BasePreviewOptions & {
  mode: UnityPreviewMode | null
  disableLoader: boolean
  username: string | null
}
```

Reply:
I added the username into schemas